hack/build/docker/builder/Dockerfile

```diff
@@ -22,7 +22,9 @@ RUN \
     go install ./... ) && \
     ( go get -d github.com/mattn/goveralls && \
     cd $GOPATH/src/github.com/mattn/goveralls && \
-    go install ./... )
+    go install ./... ) && \
+    ( go get github.com/onsi/gomega && \
+    go get github.com/onsi/ginkgo/ginkgo )
```

Comment:
Nearly a duplicate of L13 - can you instead alter L13 and add  `go get github.com/onsi/gomega` it?